### DISCUSSION TITLE
V2rayU彻底卸载方法

### DISCUSSION THREAD

**Discussion by yanue** [yanue/V2rayU#907] [Q&A]
V2rayU彻底删除方法: (推荐使用AppCleaner)

https://github.com/yanue/V2rayU/blob/master/README.md#%E5%BD%BB%E5%BA%95%E5%8D%B8%E8%BD%BD

### 第一步: 卸载V2rayU.app
   停止v2ray-core, 然后卸载V2rayU.app

###  第二步: 彻底删除相关文件及服务
**命令**
```
cd ~/Library/LaunchAgents/
/bin/launchctl remove yanue.v2rayu.v2ray-core
/bin/launchctl remove yanue.v2rayu.http
/bin/launchctl remove net.yanue.V2rayU.Launcher

rm -f ~/Library/LaunchAgents/yanue.v2rayu.v2ray-core.plist
rm -f ~/Library/Preferences/net.yanue.V2rayU.plist
rm -f ~/Library/Logs/V2rayU.log


rm -fr ~/.V2rayU/
```

**重点**
   主要是使用launchctl命令停止yanue.v2rayu.v2ray-core服务(会自动启动)

**注意还原网络设置**

  非正常停止V2rayU可能会导致网络设置代理,卸载后网络代理没变,可能会导致无法上网
  此时需要手动欢迎网络设置

**Comment by Xuzan9396** [comment]
下载了2.3.1 现在是完全用不了了。
20/09/03 18:03:05 [Info] [3725155196] v2ray.com/core/app/proxyman/inbound: connection ends > v2ray.com/core/proxy/socks: connection ends > context canceled
2020/09/03 18:03:05 [Info] [3725155196] v2ray.com/core/app/proxyman/outbound: failed to process outbound traffic > v2ray.com/core/proxy/vmess/outbound: connection ends > context canceled
2020/09/03 18:03:19 [Info] [2873896151] v2ray.com/core/app/proxyman/inbound: connection ends > v2ray.com/core/proxy/socks: connection ends > context canceled
2020/09/03 18:03:19 [Info] [2873896151] v2ray.com/core/app/proxyman/outbound: failed to process outbound traffic > v2ray.com/core/proxy/vmess/outbound: connection ends > context canceled
2020/09/03 18:05:27 [Info] [2022727208] v2ray.com/core/proxy/socks: TCP Connect request to tcp:mtalk.google.com:5228
2020/09/03 18:05:27 [Info] [2022727208] v2ray.com/core/app/dispatcher: default route for tcp:mtalk.google.com:5228
2020/09/03 18:05:27 [Info] [2022727208] v2ray.com/core/transport/internet/tcp: dialing TCP to tcp:cn1.shadowsky.club:11081
2020/09/03 18:05:27 tcp:127.0.0.1:53867 accepted tcp:mtalk.google.com:5228 [proxy] 
2020/09/03 18:05:27 [Info] [2022727208] v2ray.com/core/proxy/vmess/outbound: tunneling request to tcp:mtalk.google.com:5228 via tcp:cn1.shadowsky.club:11081
2020/09/03 18:05:27 [Info] [1061962740] v2ray.com/core/app/proxyman/outbound: failed to process outbound traffic > v2ray.com/core/proxy/vmess/outbound: connection ends > context canceled
2020/09/03 18:05:27 [Info] [1061962740] v2ray.com/core/app/proxyman/inbound: connection ends > v2ray.com/core/proxy/socks: connection ends > v2ray.com/core/proxy/socks: failed to transport all TCP response > io: read/write on closed pipe

**Comment by Elegeion** [comment]
求救找不到Core这个问题怎么解决

[a comment by an automated account is not shown]

**Reply by ZhiquanW** [reply]
同志们，这个方法可行。感谢活菩萨提供的方法！

**Reply by matthew188** [reply]
这玩意太顽强了，已经删除了，根目录下也都搜索不到了，还是一直在弹窗，登录项上居然还能显示。好在关闭后不弹窗了。

**Comment by Elegeion** [comment]
你在威胁我？




------------------&nbsp;原始邮件&nbsp;------------------
发件人: ***@***.***&gt;; 
发送时间: 2023年7月7日(星期五) 晚上6:16
收件人: ***@***.***&gt;; 
抄送: ***@***.***&gt;; ***@***.***&gt;; 
主题: Re: [yanue/V2rayU] V2rayU彻底卸载方法 (Discussion #907)





 
我已经试了issues里的所有办法，包括强制签名，重装，用CleanMyMac卸载，按上面步骤卸载，都无法屏蔽《“V2RayU•app”将对你的电脑造成伤害。你应该将它移到垃圾篓。》的无限弹窗骚扰！！！！！
 请帮帮我😭
 我使用Inter i5的MacBook pro，运行Ventura 13.4系统

[a comment by an automated account is not shown]

**Comment by Yukielz** [comment]
还需要增加这一条
`/bin/launchctl remove net.yanue.V2rayU.Launcher`
